commands_to_load/Administration.py

```diff
@@ -11,18 +11,29 @@ class Administration():
 	def __init__(self, bot):
 		self.bot = bot
 
+	def realCog(self, cog):

```

Comment:
this is very much up to you but it might make more sense to name it `validCog`? that might it a more logical name. I will approve either way but just wanted to pitch in that opinion